src/bci_build/package/pulseaudio/client.conf

```diff
@@ -0,0 +1,2 @@
+autospawn = no
```

Comment:
the files in src/bci_build/package/pulseaudio/* are probably part of kiosk-firefox not of the xorg container though?

Reply:
messed up the cherry-pick :facepalm: